client/lib/url/index.js

```diff
@@ -20,4 +21,4 @@ function isExternal( url ) {
 	return isOutsideCalypso( url ) && ! startsWith( url, '//wordpress.com' );
 }
 
-export default { isOutsideCalypso, isExternal };
+export default { isOutsideCalypso, isExternal, support };
```

Comment:
I guess my initial intention to make something similar to how lodash works. I have removed exporting from `lib/url/index`.